milestone-3/backend/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon index.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@types/express": "^4.17.15",
+    "@types/mongoose": "^5.11.97",
+    "mongoose": "^6.7.5",
+    "ts-node": "^10.9.1"
+  }
```

Comment:
It looks like you only installed the types for express but not express itself, can you install express so that everything runs ok?

Reply:
Got it, just installed express in my latest commit. Thanks for letting me know!